controller-api/src/main/java/com/yahoo/vespa/hosted/controller/api/integration/deployment/JobType.java

```diff
@@ -155,7 +162,7 @@
                             Map.of(main, ZoneId.from("perf"   , "us-east-3")));
 
     private final String jobName;
-    private final Map<SystemName, ZoneId> zones;
+    final Map<SystemName, ZoneId> zones;
```

Comment:
Why?

Reply:
Package private is OK. Access it in test.